pkg/tsdb/elasticsearch/response_parser.go

```diff
@@ -151,6 +151,17 @@ func processLogsResponse(res *es.SearchResponse, target *Query, configuredFields
 			}
 		}
 
+		// we are going to add an `id` field with the concatenation of `_id` and `_index`
+		_, ok := doc["id"]
+		if !ok {
+			_, okId := doc["_id"]
+			_, okIndex := doc["_index"]
+			if okId && okIndex {
+				doc["id"] = fmt.Sprintf("%v#%v", doc["_index"], doc["_id"])
+				propNames["id"] = true
+			}
+		}
+
```

Comment:
Could we move this up above so we don't have to do `propNames["id"] = true`?

```
for key := range doc {
			propNames[key] = true
		}
```

Reply:
Also I know I suggested check bellow, but I think we don't need it cause it is always going to be ok, as we are creating it couple lines above. So I don't think we actually need it cause it is always going to be `true`. 

```
			_, okId := doc["_id"]
			_, okIndex := doc["_index"]
```

Reply:
Done in 6c39472cec510bc7fcc1fed6cfda7be4d8f5ba4d